Dockerfile

```diff
@@ -23,6 +23,24 @@ RUN cd /opt/android-ndk-tmp && mv ./android-ndk-r12 /opt/android-ndk
 RUN rm -rf /opt/android-ndk-tmp
 # add to PATH
 ENV PATH ${PATH}:${ANDROID_NDK_HOME}
+
+
+# ------------------------------------------------------
+# --- Android CMake
+
+# download
+RUN mkdir /opt/android-cmake-tmp
+RUN cd /opt/android-cmake-tmp && wget -q https://dl.google.com/android/repository/cmake-3.6.3155560-linux-x86_64.zip -O android-cmake.zip
+# uncompress
+RUN cd /opt/android-cmake-tmp && unzip -q android-cmake.zip -d android-cmake
+# move to it's final location
```

Comment:
if you're around feel free to fix this at https://github.com/bitrise-docker/android-ndk/pull/20/files#diff-3254677a7917c6c01f55212f86c57fbfR20 too :D